client/stylesheets/_toCat.scss

```diff
@@ -4239,16 +4238,16 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 .profileReceivedCardLeft {
-    width: 45% !important;
+    width: 35% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces